gamestonk_terminal/portfolio/portfolio_optimization/optimizer_model.py

```diff
@@ -87,6 +87,7 @@ def get_maxsharpe_portfolio(
     """
     stock_prices = yahoo_finance_model.process_stocks(stocks, period)
     ef = prepare_efficient_frontier(stock_prices)
+    print(rfrate)
```

Comment:
So we want to avoid printing in _model.  So if you want to print the rfrate, do it in view (which you actually have already)

Reply:
This is only to show that the rate matches so I can diagnose the error. Will be removed once the error is gone.